content/docs/5.1/develop/clients/introduction.md

```diff
@@ -24,6 +24,6 @@ TiKV has clients for a number of languages:
 | -------------------------- | ----------------- | ----------------- | ---------------------- |
 | [Java Client](../java)     | Stable            | Under development | >= 2.0.0               |
 | [Go Client V2](../go)      | Unstable          | Unstable          | >= 5.0.0               |
```

Comment:
Is it appropriate to specially mark the version of go client?

Reply:
done